openedx/features/badging/tests/test_views.py

```diff
@@ -69,8 +66,9 @@ def test_trophycase_with_some_earned_badges(self, mock_populate_trophycase):
         UserBadgeFactory(user=any_other_user, course_id=CourseKeyField.Empty, badge=badge2)
 
         mock_populate_trophycase.return_value = dict()
-        badging_views.trophycase(request)
-        mock_populate_trophycase.assert_called_once_with(request.user, mock.ANY, [user_badge])
+        response = self.client.get(reverse('trophycase'), follow=True, data= {'json': True})
```

Comment:
please remove extra space `data={'json': True}`.

Reply:
Are we expecting any chain of redirects, if not then why are we passing `follow=True`?

Reply:
Removed the space.
In absence of follow=Ture we end up on 302 found.